src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -100,6 +118,55 @@ internal static string GetAccountId(string functionArn)
             return null;
         }
 
+        internal static string GetFaasTrigger<TInput>(TInput input)
+        {
+            var trigger = "other";
+            if (input is APIGatewayProxyRequest || input is APIGatewayHttpApiV2ProxyRequest)
+            {
+                trigger = "http";
+            }
+
+            return trigger;
+        }
+
+        internal static IEnumerable<KeyValuePair<string, object>> GetFunctionDefaultTags<TInput>(TInput input, ILambdaContext context)
+        {
+            var tags = new List<KeyValuePair<string, object>>
+            {
+                new(AWSLambdaSemanticConventions.AttributeFaasTrigger, GetFaasTrigger(input)),
+            };
+
+            var functionName = GetFunctionName(context);
+            if (functionName != null)
+            {
+                tags.Add(new(AWSLambdaSemanticConventions.AttributeFaasName, functionName));
+            }
+
+            if (context == null)
+            {
+                return tags;
+            }
+
+            if (context.AwsRequestId != null)
+            {
+                tags.Add(new(AWSLambdaSemanticConventions.AttributeFaasExecution, context.AwsRequestId));
+            }
+
+            var functionArn = context.InvokedFunctionArn;
+            if (functionArn != null)
+            {
+                tags.Add(new(AWSLambdaSemanticConventions.AttributeFaasID, functionArn));
```

Comment:
This should be stored in `aws.lambda.invoked_arn` https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-lambda.md#all-triggers

For the faas.id, you need to make sure that any component of the ARN after the function name (alias or version) is removed (spec currently says to add the faas.version, as read from envvar, instead but I don't think this makes much sense actually)

Reply:
Now, the 8th part (version) of arn and everything after that is removed. I added corresponding doc references in the method removing parts.